android/quest/src/main/assets/configs/app/application_config.json

```diff
@@ -2,14 +2,16 @@
   "appId": "app",
   "configType": "application",
   "theme": "DEFAULT",
-  "appTitle": "eCBIS",
+  "appTitle": "eCBIS (CHA & CHSS)",
   "remoteSyncPageSize": 100,
   "languages": [
     "en"
   ],
   "useDarkTheme": false,
   "syncInterval": 30,
-  "syncStrategies": ["Organization", "Location", "CareTeam", "Practitioner"],
+  "syncStrategy": [
+    "Location"
+  ],
```

Comment:
Syncing by Location syncs all data from staging server. Sync has not been optimized yet so this takes over 50mins to complete; not ideal for development.

Reply:
> Should we create new configurations for the supervisor app? As long as the app Id and name of the directory are different it is okay we can still load them from assets. Or is it fine to mix the configs and use the same app debug id?

- My thinking has been we use the config directory to hold different variants of the modules we have. I am okay with all being in the same place. We have however separated them on the fhir-resource repo

Reply:
> Syncing by Location syncs all data from the staging server. Sync has not been optimized yet so this takes over 50mins to complete; not ideal for development.

Makes sense. Let update it